acquia_lift.module

```diff
@@ -813,191 +707,24 @@ function acquia_lift_personalize_option_set_save($option_set) {
-  // paused is if there are no more option sets.
-  if (count($new_option_sets) == 0) {
-    personalize_pause_if_running($agent->machine_name);
-  }
-}
-
-/**
- * Makes sure all option set changes are sync'd to Acquia Lift.
- *
- * @param $agent
- *   The agent whose option sets have changed.
- * @param $old_option_sets
- *   THe old option sets that Acquia Lift knew about.
- * @param $new_option_sets
- *   The new option sets.
- */
-function acquia_lift_sync_option_sets($agent, $old_option_sets, $new_option_sets, $sync_only_on_changes = FALSE) {
-  if ($agent_instance = personalize_agent_load_agent($agent->machine_name)) {
-    if (!$agent_instance instanceof AcquiaLiftAgentInterface) {
-      return;
-    }
-    try {
-      // Tell Acquia Lift about the change to the decision structure.
-      $old_decisions = AcquiaLiftAgent::convertOptionSetsToDecisions($old_option_sets);
-      $new_decisions = AcquiaLiftAgent::convertOptionSetsToDecisions($new_option_sets);
-      if (!$sync_only_on_changes || ($old_decisions != $new_decisions)) {
-        $agent_instance->syncDecisions($old_decisions, $new_decisions);
-      }
 
-      // Save the agent data to the db.
-      $agent->data['decisions'] = $new_option_sets;
-      personalize_agent_save($agent);
-    }
-    catch (Exception $e) {
-      drupal_set_message($e->getMessage(), 'error');
-    }
-  }
-}
-
-/**
- * Syncs fixed targeting rules to Acquia Lift.
- * @param $agent
- * @param $new_option_sets
- */
-function acquia_lift_sync_fixed_targeting($agent, $new_option_sets) {
-  if ($agent_instance = personalize_agent_load_agent($agent->machine_name)) {
-    if (!$agent_instance instanceof AcquiaLiftAgentInterface) {
-      return;
-    }
-    try {
-      $agent_instance->syncFixedTargeting($new_option_sets);
-    }
-    catch (Exception $e) {
-      drupal_set_message($e->getMessage(), 'error');
-    }
-  }
-}
-
-/**
- * Implements hook_personalize_goal_save().
- */
-function acquia_lift_personalize_goal_save($goal_info) {
-  $agent = personalize_agent_load($goal_info['agent']);
-  if (!acquia_lift_is_testing_agent($agent)) {
-    return;
-  }
-  $old_goals = isset($agent->data['goals']) ? $agent->data['goals'] : array();
-  $current_goals = personalize_goal_load_by_conditions(array('agent' => $agent->machine_name));
-  foreach ($current_goals as $goal) {
-    $new_goals[$goal->action] = $goal->value;
-  }
-  // For the current goal, take the info from what was passed in, rather
-  // than from the db.
-  $new_goals[$goal_info['action']] = $goal_info['value'];
-  acquia_lift_sync_goals($agent, $old_goals, $new_goals, TRUE);
-  // This agent will need to be re-verified as the changes may have changed
-  // whether or not it is valid.
-  acquia_lift_agent_clear_verified_status($goal_info['agent']);
 }
 
 /**
  * Implements hook_personalize_goal_delete().
  */
 function acquia_lift_personalize_goal_delete($goal_info) {
```

Comment:
Yep, will remove - if there's anything that needs to be done here I can tackle it in the other story